server/selection.go

```diff
@@ -135,12 +135,15 @@ func (s *MinLSSelector) Select(ctx context.Context) *BroadcastSession {
 		return s.selectUnknownSession(ctx)
 	}
 
-	minSess := sess.(*BroadcastSession)
-	if minSess.LatencyScore > s.minLS && len(s.unknownSessions) > 0 {
-		return s.selectUnknownSession(ctx)
+	lowestLatencyScoreKnownSession := heap.Pop(s.knownSessions).(*BroadcastSession)
```

Comment:
What if there's another session on the heap? I know this is how it was before, so just trying to understand the logic here

Reply:
I think it works like this: 
- take the first session from the heap
- if no more sessions, then refresh the sessions

So, it's no a problem if there are more than 1 sessions on the heap, we just pop the fist one.